Python/Product/Analysis/LanguageServer/Server.Completion.cs

```diff
@@ -22,10 +22,7 @@
 
 namespace Microsoft.PythonTools.Analysis.LanguageServer {
     public sealed partial class Server {
-        public override async Task<CompletionList> Completion(CompletionParams @params) {
-            await _analyzerCreationTask;
-            await IfTestWaitForAnalysisCompleteAsync();
-
+        public override Task<CompletionList> Completion(CompletionParams @params) {
```

Comment:
For the record, I disagree with the warning that says not to use `async` when you aren't using `await` (and it's disabled explicitly). We've had 100% more bugs due to returning `null` instead of a Task than we've had issues with any performance overhead.

Reply:
Yeah, it is annoying. I'll try to suppress it in .NET Core projects too then.